src/components/Validator.js

```diff
@@ -71,6 +71,15 @@ export const Validator = {
         if (!FormioUtils.boolValue(setting)) {
           return true;
         }
+
+        // WYSIWYG validation
+        if (component.quill) {
```

Comment:
This should not be here. You should implement this in the ```isEmpty``` method inside of the ```TextAreaComponent``` class. This will keep out component specific logic out of the validator.